openassessment/xblock/test/test_staff_area.py

```diff
@@ -770,7 +770,7 @@ def test_staff_delete_student_state(self, xblock):
         self.assertIn("final grade", resp.body.lower())
 
         # Clear the submission
-        xblock.clear_student_state('Bob', 'test_course', xblock.scope_ids.usage_id)
+        xblock.clear_student_state('Bob', 'test_course', xblock.scope_ids.usage_id, STUDENT_ITEM['student_id'])
```

Comment:
Yes, in this instance the test is specifically checking "Can a staff user clear state on themselves".

Since `clear_student_state` has (after much debate) been allowed to proceed as an unbound xblock method call, the xblock has no way of verifying that the user passed in is staff; the assumption is that the runtime has secured any code that will call this method. Since our test is acting as the runtime, the simplest test is to use the staff user `Bob` we've already created.